src/data/knownFunctions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "setOwner(address)": "Set `$1` as the new owner",
+    "setOwner(bytes32,address)": "Set `$2` as the new owner of the `$1` node",
+    "transfer(address,address,uint256)": "Transfer `@tokenAmount($1, $3)` to `$2`"
```

Comment:
Not generally against this idea, but find it a bit unnecessary.

Instead of re-evaluating radspec, we could just run a javascript function here with the arguments to build the string. We could inject the web3 instance as well, if we really wanted to allow web3 calls.